tasks/secrets.yml

```diff
@@ -3,10 +3,22 @@
   file:
     dest: "{{ _run_temp_dir.path }}/src/{{ item.path | dirname }}"
     state: directory
-  loop: "{{ pantheon_deploy.secrets | default([]) }}"
+  loop: "{{ pantheon_deploy.secrets  | default([]) }}"
+  when: pantheon_deploy.secrets.json is undefined
```

Comment:
If we use a top level key here for `files`, we woudn't need to add the conditional. Instead, we'd change the loop to `pantheon_deploy.secrets.files | default([])` and it would have the same effect.

Reply:
Changed and updated!